modules/system/console/MixCompile.php

```diff
@@ -180,7 +180,9 @@ protected function createCommand(string $mixJsPath): array
         $command = $this->argument('webpackArgs') ?? [];
         array_unshift(
             $command,
-            $basePath . '/node_modules/webpack/bin/webpack.js',
+            "npx",
+            "webpack",
+            "build",
```

Comment:
Why is this required?

Reply:
Windows won't run scripts in powershell without correct privilidges. Rather than loading the webpack script directly, running it with npx will solve this issue. However without definition of "npx" in %path% environment variable, it may cause running issue.